korgan/professional_consultation_guard.py

```
@@ -0,0 +1,344 @@
+        if f"/{adilet_id}" in path:
+            return act_id
+    return ""
+
+
+def _normalize_quote(text: str) -> str:
+    value = str(text or "").replace("ё", "е").replace("Ё", "Е").lower()
+    value = re.sub(r"\s+", " ", value)
+    return value.strip(" \t\r\n.;,«»\"")
+
+
+def _corpus_article_check(article: str, source_url: str, provision_text: str) -> bool:
```

Ruff (0.16.2)

[warning] Too many return statements (7 > 6)

(PLR0911)